de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/pricelist/report_bpblock.jrxml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Jaspersoft Studio version 6.5.1.final using JasperReports Library version 6.5.1  -->
```

Comment:
Is still  need this report?
Can't you use the general one?

Reply:
Using the general one which has a much higher height, a blank space appears on new pages for new bpartners. As the general one is only used in the page title where there is enough blank space until the report starts, that is not a problem there, however in this case it is.